components/underlay/stories/underlay.stories.js

```diff
@@ -33,7 +33,6 @@ export default {
 };
 
 export const Default = Template.bind({});
-Default.tags = ["autodocs", "!test"];
```

Comment:
Hm, why did we have `!test` for this default? Wouldn't that remove it from Chromatic? I see an [underlay snapshot from main](https://www.chromatic.com/component?appId=64762974a45b8bc5ca1705a2&csfId=components-underlay&buildNumber=2477&k=66a27c9ae1a436a8461a65f7-Context%3A+Express-interactive-true&h=48&b=-1) though (I think) from a while back. Just curious 😊

Reply:
It wasn't doing anything previously. I'm not sure why it was there.